checking.md

```diff
@@ -3,49 +3,58 @@
 # Checkers
 
 - Getting set up as a Checker
-  - Talk to your DataRescue guide to make sure you can become a Checker.
-    - In general, Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
-  - Get set up with the application http://drp-upload-checker.herokuapp.com
-    - Credentials and other details will be provided by the organizers.
-  - Contact your DataRescue guide, if you need any assistance.
+  - Apply to become a Checker 
+    - By asking your DataRescue guide or by filling out [this form](XXX) 
+    - Skills recommended: in general, Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
+    - Note that a Gmail email address is required to apply.
+    - Note also that you should be willing to have your real name be associated with the datasets, to follow archival best practices (see [Trust and archival best practices](XXX) for more information).
+  - Credentials, slack invite, Uncrawlable spreadsheet URL, and other details will be provided once your application is approved.
+  - Test the Uploader application http://drp-upload.herokuapp.com with the credentials provided
```

Comment:
Make this link clickable? Might be fine as is, but would be good to have as a []()